include/deal.II/base/bounding_box.h

```diff
@@ -24,6 +24,39 @@
 DEAL_II_NAMESPACE_OPEN
 
 /**
+ * The enumerator NeighborType describes the neighboring relation between
+ * two bounding boxes.
+ */
+enum class NeighborType
+{
+  /**
+   * value 0: not_neighbours i.e. the intersection is empty
```

Comment:
you don't need to say what value is there, so i would drop `value x:` here and below. Users shall not see any difference if one day you switch to value 999, 998, etc.

Reply:
also remove `_` here and below, the names read fine without it.